### ISSUE TITLE
Can datatable display be customized?

### ISSUE THREAD

**Issue by bauchetg** [solgenomics/sgn#1584]
Expected Behavior 
--------------------------------------------------------------------------

For searches (stocks, traits, trials) It would be very useful to leave the user ability to select information fields  to display (i.e. curator, institution, species, ploidy level).
For the trait search, we could have the ability to select the ontology you wish to search in.

For Bugs:
---------

**Comment by bauchetg** [comment]
@naama I created a patch to add "banana trait set" as prop see branch topic/add_cvterm prop2trait
AddBananaOntologyProps.pm.
After review with @nickmorales, it seems a better approach to use current prop "trait property" and add it a value to it like "trait set type" and a patch for that trait.
This way we can load information set through a loading script, to avoid changing the current query.
This data and other props could be displayed on the different searches (trait but also trial/prop or crosses).
@nickmorales: Rony approved a view mode from the search/trait page where we could subset using a list as we discussed
